auth/presentation/src/main/java/com/sergius/auth/presentation/login/SignInScreenRoot.kt

```diff
@@ -42,6 +45,7 @@ fun SignInScreenRoot(
     onSignUpClick: () -> Unit,
     viewModel: LoginViewModel = koinViewModel(),
 ) {
+    viewModel.isComposing.collectAsStateWithLifecycle()
```

Comment:
You can collect the `state` variable here instead of `isComposing`, and directly pass the required fields like `state.canLogin` and `state.isLoggingIn`. This keeps your UI reactive to all relevant state changes without needing multiple individual flows.